crates/shared_types/src/directive.rs

```diff
@@ -276,6 +302,42 @@ impl Directive {
         ))
     }
 
+    fn parse_attr_string_list_directive<'a, E: ParseError<&'a str> + ContextError<&'a str>>(
```

Comment:
I'd rather support something like `attr_list:foo:["bar", "baz", "quux"]` so we can give the literal list as it will appear in the code. I think it could be as simple as starting with `[` and ending at `]` as long as it is not inside a quote. I know that means you have to parse the strings which is non-trivial... but I mean, having an escape hatch that only works for very narrow cases seems like it just sets us up to revisit.

Reply:
Changed to:

```
attr.string_list: plugins -> { //x/y/z:artifact, bar }
```